examples/param/param.js

```diff
@@ -11,7 +11,9 @@ const endpoint = __ENV.ENDPOINT || "otel-collector:4317"
 const client = new tracing.Client({
     endpoint,
     exporter: tracing.EXPORTER_OTLP,
-    insecure: true,
+    tls: {
+      insecure: false,
```

Comment:
The examples don't use tls. They are also used in the Tempo docker-compose examples where we also rely on the `insecure: true` setting

```suggestion
      insecure: true,
```

Reply:
Ah yes of course :)